pages/common/sails.md

```diff
@@ -0,0 +1,19 @@
+sails
+
+> Sails.js is a realtime enterprise level MVC framework built on top of Node.js.
+
+- Start Sails
+
+`sails lift`
+
+- Create new Sails project
+
+`sails new {{projectName}}`
+
+- Display the version
+
+`sails -v` or `sails --version`
```

Comment:
This is trivial. I would probably avoid writing this example. Unless it is java command, which has `-version` (single dash).